requirements/requirements-pytorch.txt

```diff
@@ -1,7 +1,7 @@
 torch>=1.9,<3
-lightning>=2.2.2,<2.5
+lightning>=2.2.2,<2.6
 # Capping `lightning` does not cap `pytorch_lightning`, so we cap manually
-pytorch_lightning>=2.2.2,<2.5
+pytorch_lightning>=2.2.2,<2.6
```

Comment:
Training seems to fail with `torch-2.8`, checking if this is due to lightning incompatibility.